src/tetris/profiler/service/faceAnalysisService.ts

```diff
@@ -0,0 +1,72 @@
+import BaseService from "tetris/profiler/service/baseService";
+import Measurement from "tetris/profiler/measurement/measurement";
+import CameraController from "tetris/profiler/hardwareController/cameraController";
+import HardwarePermission from "tetris/profiler/hardwareController/hardwarePermission";
+
+
+export default class FaceAnalysisService extends BaseService {
+	//region public members
+	public static get serviceName(): string {
+		return 'FaceAnalysisService';
+	}
+	//endregion
+
+	//region public methods
+	//endregion
+
+	//region constructor
+	public constructor(errorCallback: (senderName: string, error: Error) => void) {
+		super(FaceAnalysisService.serviceName, errorCallback);
+	}
+	//endregion
+
+	//region private members
+	private _successCallback: (senderName: string, measurement: Measurement<Object>) => void;
+	//endregion
+
+	//region private methods
+	private static get _faceplusplusApiKey(): string {
+		return '#API_KEY_REQUIRED';
+	}
+
+	private static get _faceplusplusApiSecret(): string {
+		return '#API_SECRET_REQUIRED';
+	}
+
+	private async _requestFaceAnalysis(image: string): Promise<void> {
+		const formData = new FormData();
+		const url = 'https://proxyboy2.herokuapp.com/fpp/detect';
+
+		// Prepare form data
+		formData.append("api_key", FaceAnalysisService._faceplusplusApiKey);
```

Comment:
Just curious: why not `this._faceplusplusApiKey`?

Reply:
Because  `_faceplusplusApiKey` is a static method and can be called on the class directly. `this._faceplusplusApiKey` would call the method on an instance of `FaceAnalysisService` and bind this, which is unnecessary for this function.